easybuild/easyconfigs/v/vcfnp/vcfnp-2.3.0_numpy_setup.patch

```diff
@@ -0,0 +1,20 @@
+diff -ru vcfnp-2.3.0/setup.py vcfnp-2.3.0.fixed/setup.py
```

Comment:
please add a descriptive comment including author

Reply:
you'll also need to update the patch checksum :)

Reply:
ups yes, fixed it now